packages/core/helper-plugin/src/features/AppInfo.tsx

```diff
@@ -1,44 +1,35 @@
-import * as React from 'react';
+import { Context, ReactNode, createContext, useContext, useMemo } from 'react';
 
-import PropTypes from 'prop-types';
+type EmptyObject = Record<string, never>;
+
+type AppInfoContextValue = {
+  autoReload?: boolean;
+  communityEdition?: boolean;
+  currentEnvironment?: string;
+  dependencies?: Record<string, string>;
+  latestStrapiReleaseTag?: string;
+  nodeVersion?: string;
+  // projectId?: string;
```

Comment:
Why have you commented all this out everywhere?

Reply:
I left a comment above about a comment asking for them to be reviewed and I not being able to see `projectId` being used

Reply:
ah i see, lets leave it for now, because it looks like we're doubling down on uuid vs projectId which i think at the root have the same value 🤔 thank you for bringing it up though :)

Reply:
no problem, I'll uncomment it.